assets/dev/js/utils/time.js

```diff
@@ -16,12 +16,15 @@ export default function getUserTimestamp() {
 	UTCTimestamp = UTCTimestamp.slice( 0, -1 );
 
 	// Create the offset string in the format `+HH:00` (or minus (-) prefix for negative offset instead of plus)
-	const timezoneOffsetInHours = timezoneOffset / 60,
+	const decimalTimezoneOffset = timezoneOffset / 60,
 		// Negative offsets include a '-' sign in the getTimezoneOffset value, positive values need a '+' prefix (ISO8601).
-		sign = 0 <= timezoneOffsetInHours ? '+' : '',
-		addedZero = 10 < ( timezoneOffsetInHours ) ? '0' : '';
+		sign = 0 <= decimalTimezoneOffset ? '+' : '-',
+		hours = Math.abs( Math.floor( decimalTimezoneOffset ) ),
+		minutes = Math.abs( decimalTimezoneOffset % 1 ) * 60,
+		addZeroToHour = 10 > ( hours ) ? '0' : '',
+		addZeroToMinutes = 10 > ( minutes ) ? '0' : '';
 
-	const formattedTimezoneOffset = sign + addedZero + timezoneOffsetInHours + ':00';
+	const formattedTimezoneOffset = sign + addZeroToHour + hours + ':' + addZeroToMinutes + minutes + ':00';
```

Comment:
```suggestion
	const formattedTimezoneOffset = sign + addZeroToHour + hours + ':' + addZeroToMinutes + minutes;
```

The `:00` represented minutes, so now that you are adding the minutes with variables, this suffix is redundant.